dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ConfigurationController.java

```diff
@@ -399,4 +403,10 @@ public void setCorsWhitelist( @RequestBody String input )
     {
         return config.isReadOnlyMode();
     }
+
+    @RequestMapping( value = "/appHubUrl", method = RequestMethod.GET )
+    public @ResponseBody String getAppHubUrl( Model model, HttpServletRequest request )
+    {
```

Comment:
If `AppHub` is the new official term, could we rename the method `AppManager.getAppStoreUrl` to `getAppHubUrl` ?

Reply:
Yes it should be named apphub through the entire code base, I was thinking the full code base rename could be in a separate PR but maybe its just better to get it done all at once. I can have a look, since it would be affecting configuration (dhis.conf) and lots of other places as well.

Reply:
Done.